src/components/sbb-journey-summary/sbb-journey-summary.scss

```diff
@@ -0,0 +1,52 @@
+.journey-summary {
+  padding: var(--sbb-spacing-responsive-s);
+  display: flex;
+  flex-direction: column;
+}
+
+.journey-summary__via-block {
+  display: flex;
+
+  .journey-summary__via-text {
+    padding-right: var(--sbb-spacing-fixed-1x);
```

Comment:
we would use here padding-inline-end

`padding-inline-end: var(--sbb-spacing-fixed-1x)`